QuizApp/App/Screens/Quiz/QuizListViewModel.swift

```diff
@@ -1,9 +1,11 @@
 import Foundation
+import Combine
 
 final class QuizListViewModel {
 
-    var categories: [Category] = [.all, .sport, .movies, .music, .geography]
-    @Published var quizes: [QuizViewModel] = []
+    var categories: [CategoryViewModel] = [.all, .sport, .movies, .music, .geography]
+    @Published var sections: [CategoryViewModel: [QuizViewModel]] = [:]
```

Comment:
No need to store sections. You can extract everything from categories and quizzes.

This dictionary will only be used when you are creating sections and putting items in them.